valintalaskenta-tulos-service/src/main/java/fi/vm/sade/valintalaskenta/tulos/service/impl/ValintalaskentaTulosServiceImpl.java

```diff
@@ -186,19 +188,19 @@ private void applyMuokatutJonosijatToHakukohde(String hakuOid, List<HakukohdeDTO
 
   private void applyMuokatutJonosijat(
       String hakukohdeoid,
-      List<ValintatietoValinnanvaiheDTO> b,
-      List<MuokattuJonosija> a,
-      List<HarkinnanvarainenHyvaksyminen> c) {
-    for (ValintatietoValinnanvaiheDTO dto : b) {
+      List<ValintatietoValinnanvaiheDTO> valinnanvaiheet,
+      List<MuokattuJonosija> muokatutJonosijat,
+      List<HarkinnanvarainenHyvaksyminen> harkinnanvaraisetHyvaksymiset) {
+    for (ValintatietoValinnanvaiheDTO dto : valinnanvaiheet) {
       for (ValintatietoValintatapajonoDTO valintatapajonoDTO : dto.getValintatapajonot()) {
```

Comment:
Tässä mammuttiurakassa ei varmaan ole nyt ollut paukkuja siistimis-refaktorointeihin mutta ehkä 4 sisäkkäisen for-loopin modausta voisi harkita, tyyliin sisimmäinen apumetodiin, ja for-hässäkän korvausta streameilla map/flatmap